ahnlich/ai/src/cli/server.rs

```diff
@@ -29,47 +43,105 @@ pub enum Commands {
     SupportedModels(SupportedModelArgs),
 }
 
+static DEFAULT_CONFIG: OnceLock<AIProxyConfig> = OnceLock::new();
+
 #[derive(Args, Debug, Clone)]
 pub struct AIProxyConfig {
     /// Ahnlich AI proxy port
-    #[arg(long, default_value_t = 1370)]
+    #[arg(long, default_value_t =
+    DEFAULT_CONFIG.get_or_init(AIProxyConfig::default).port.clone())]
```

Comment:
Some of these things have been moved to a shared config which we then called `common` on main and we share it between DB and AI

Reply:
Yes, that was also refactored.